frmts/jp2kak/jp2kakdataset.cpp

```diff
@@ -2243,10 +2251,76 @@ JP2KAKCreateCopy( const char * pszFilename, GDALDataset *poSrcDS,
 
         CPLDebug("JP2KAK", "layer_bytes[] = %g\n",
                  static_cast<double>(layer_bytes[layer_count - 1]));
-    }
-    else
-    {
-        bReversible = true;
+    } else {
+        // Use RATE option.
+        char** dRateStr = CSLTokenizeStringComplex(CSLFetchNameValue(papszOptions,"RATE"),
+                                                   ",", FALSE, FALSE );
+        int rate_count = CSLCount( dRateStr );
+        if(rate_count <= 0) {
+            CPLError(CE_Fatal, CPLE_AppDefined,
+                     "RATE argument must be followed by a string identifying one or more bit-rates, separated by commas.\n");
```

Comment:
termination \n not needed
```suggestion
                     "RATE argument must be followed by a string identifying one or more bit-rates, separated by commas.");
```

Reply:
Fix in 1e0a4f0561cc4b01ce861a8f933b799babdfff8b